fastgen/configs/methods/config_anyflow.py

```
@@ -0,0 +1,96 @@
+    # Weight of the co-trained Stage-2 flow-map loss in the student update.
+    # The reference runs it at weight 1 (cotrain_forward_kl: True); 0 disables.
+    cotrain_pretrain_weight: float = 1.0
+
+    # Rollout NFE list, sampled uniformly per iteration with rank-0 broadcast
+    # (reference rollout_cfg.num_inference_steps_list). None falls back to
+    # the fixed student_sample_steps.
+    student_sample_steps_list: Optional[List[int]] = None
+
+    # Text dropout for the co-trained flow-map loss (reference drop_text_ratio).
+    cond_dropout_prob: Optional[float] = None
+    cond_keys_no_dropout: List[str] = []
```

Ruff (0.15.15)

[warning] Do not use mutable default values for dataclass attributes

(RUF008)